WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -22,31 +17,8 @@
       expect(response.status).to eq 200
     end
 
-    it 'sorts by name, even if name is unpopulated because they registered with a WCA account' do
-      # First accept registrations
-      registration.update_column(:accepted_at, Time.now)
-      registration_without_user.update_column(:accepted_at, Time.now)
-
-      get :index, competition_id: competition
-      registrations = assigns(:registrations)
-      expect(registrations.length).to eq 2
-      expect(registrations[0].name).to eq "Aaron"
-      expect(registrations[1].name).to eq "Zzyzx"
-    end
-
-    it 'can set name, email, events, countryId for registrations without user' do
-      patch :update, id: registration_without_user.id, registration: { name: "test name", email: "foo@bar.com", countryId: "smerbia",
-                                                                       registration_events_attributes: [ {event_id: "222"} ] }
-      registration_without_user.reload
-      expect(registration_without_user.name).to eq "test name"
-      # This registration's registration_event with event_id = '333' is already in the databse and we've just added 2x2x2
-      expect(registration_without_user.events.map(&:id)).to eq %w(333 222)
-      expect(registration_without_user.email).to eq "foo@bar.com"
-      expect(registration_without_user.countryId).to eq "smerbia"
-    end
```

Comment:
With the current code I don't think anyone can register without a user id. Is that correct?